peribolos/knative-extensions.yaml

```diff
@@ -1025,8 +1025,10 @@ orgs:
           eventing-rabbitmq: write
         members:
         - ikavgo
+        - mkuratczyk
```

Comment:
Let's first add folks to the member list, have then accept their invites. Then after add folks to the rabbitmq team. 

This just avoids the churn with our machinery that updates the OWNER files across the repos.